### ISSUE TITLE
[SF_ACC][ui5-calendar]: no disabled state text for disabled year and date when using VPC mode of screen reader

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#9719]
### Bug Description

no disabled state text for disabled year and date when using VPC mode of screen reader

### Affected Component

ui5-calendar

### Expected Behaviour

In vpc mode, the disabled element should be announced as unavailable, then the screen reader user will come to know those years or dates are not available.

### Isolated Example

https://stackblitz.com/edit/github-qwnadf?file=src%2FApp.tsx

### Steps to Reproduce

1. open the demo page
2. open JAWS or NVDA
3. open VPC mode
4. check the calendar

### Log Output, Stack Trace or Screenshots

![image](https://github.com/user-attachments/assets/9b90f8b1-4391-4b23-93cd-4746d718903c)
![image](https://github.com/user-attachments/assets/4c246068-ee5d-4db8-bb60-2fda629f092c)

### Priority

High

### UI5 Web Components Version

1.24.x

### Browser

Chrome

### Additional Context

This is a high priority issue from SF customer.

### Organization

SAP SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by yanaminkova** [comment]
Hello @SAP/ui5-webcomponents-topic-b ,

Could you please review the expected screen reader behavior for the disabled items in the `ui5-calendar` component?
Thanks in advance!

Kind regards,
Yana
